src/Events/Services/SubscriptionService.cs

```diff
@@ -93,7 +93,7 @@ public async Task<ServiceError> DeleteSubscription(int id)
 
         if (!AuthorizeAccessToSubscription(subscription))
```

Comment:
Unsure about these as the issue only specifies when creating a subscription. This is for getting a subscription. @SandGrainOne might clarify